app/js/lib/view_handler/element_slide.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+var objectAssign = require( 'object-assign' ),
+
+	/**
+	 * View Handler for showing and hiding elements if the update value matches a regular expression
+	 * @class
+	 */
+	ElementSlider = {
+		el: null,
+		showOnValue: null,
+
+		// internal fields
+		slideSpeed: 600,
+		isHidden: null,
```

Comment:
> Any recommendations on how to make this intention more clear in the code?

Not really. Perhaps a brief to the point comment?

    // Null to always trigger show/hide on first call to update